cluster/hooks.go

```diff
@@ -149,7 +150,28 @@ func InstallLogging(input interface{}, param pkgCluster.PostHookParam) error {
 	if err != nil {
 		return err
 	}
-
+	if loggingParam.TLSEnabled {
+		req := &secret.CreateSecretRequest{
+			Name: "tls-for-logging-operator",
+			Type: secretTypes.TLSSecretType,
+			Tags: []string{"logging-operator"},
```

Comment:
`logging-operator` should be passed in via a variable or const to avoid code duplication.